recipe/bld.bat

```diff
@@ -175,46 +172,61 @@ if errorlevel 1 exit /b 1
 
 if "%PKG_NAME%" == "libtorch" (
     @REM Extract the compiled wheel into a temporary directory
-    if not exist "%SRC_DIR%/dist" mkdir %SRC_DIR%/dist
-    pushd %SRC_DIR%/dist
-    for %%f in (../torch-*.whl) do (
+    if not exist "%SRC_DIR%\dist" mkdir %SRC_DIR%\dist
+    pushd %SRC_DIR%\dist
+    for /f %%f in ('dir /b /S ..\torch-*.whl') do (
         wheel unpack %%f
+        if %ERRORLEVEL% neq 0 exit 1
     )
 
-    @REM Navigate into the unpacked wheel
-    pushd torch-*
+    @REM Navigate into the unpacked wheel; naming pattern of the folder is documented:
+    @REM https://github.com/pypa/wheel/blob/0.45.1/src/wheel/cli/unpack.py#L11-L12
+    pushd torch-%PKG_VERSION%
+    if %ERRORLEVEL% neq 0 exit 1
+
+    @REM Do not package `fmt.lib` (and its metadata); delete it before the move into
+    @REM %LIBRARY_BIN% because it may exist in host before installation already
+    del torch\lib\fmt.lib torch\lib\pkgconfig\fmt.pc
+    if %ERRORLEVEL% neq 0 exit 1
+    @REM also delete rest of fmt metadata
+    rmdir /s /q torch\lib\cmake\fmt
 
     @REM Move the binaries into the packages site-package directory
-    robocopy /NP /NFL /NDL /NJH /E torch\bin %SP_DIR%\torch\bin\
-    robocopy /NP /NFL /NDL /NJH /E torch\lib %SP_DIR%\torch\lib\
-    robocopy /NP /NFL /NDL /NJH /E torch\share %SP_DIR%\torch\share\
+    @REM the only content of torch\bin, {asmjit,fbgemm}.dll, also exists in torch\lib
+    robocopy /NP /NFL /NDL /NJH /E torch\lib\ %LIBRARY_BIN%\ torch*.dll c10.dll shm.dll
+    robocopy /NP /NFL /NDL /NJH /E torch\lib\ %LIBRARY_LIB%\ torch*.lib c10.lib shm.lib
+    robocopy /NP /NFL /NDL /NJH /E torch\share\ %LIBRARY_PREFIX%\share
     for %%f in (ATen caffe2 torch c10) do (
-        robocopy /NP /NFL /NDL /NJH /E torch\include\%%f %SP_DIR%\torch\include\%%f\
+        robocopy /NP /NFL /NDL /NJH /E torch\include\%%f %LIBRARY_INC%\%%f\
     )
 
     @REM Remove the python binary file, that is placed in the site-packages
     @REM directory by the specific python specific pytorch package.
-    del %SP_DIR%\torch\lib\torch_python.*
+    del %LIBRARY_BIN%\torch_python.* %LIBRARY_LIB%\torch_python.* %LIBRARY_LIB%\_C.lib
+    if %ERRORLEVEL% neq 0 exit 1
 
     popd
     popd
 
     @REM Keep the original backed up to sed later
     copy build\CMakeCache.txt build\CMakeCache.txt.orig
+    if %ERRORLEVEL% neq 0 exit 1
 ) else if "%PKG_NAME%" == "pytorch" (
+    @REM Move libtorch_python and remove the other directories afterwards.
+    robocopy /NP /NFL /NDL /NJH /E %SP_DIR%\torch\lib\ %LIBRARY_BIN%\ torch_python.dll
+    robocopy /NP /NFL /NDL /NJH /E %SP_DIR%\torch\lib\ %LIBRARY_LIB%\ torch_python.lib
+    robocopy /NP /NFL /NDL /NJH /E %SP_DIR%\torch\lib\ %LIBRARY_LIB%\ _C.lib
+    rmdir /s /q %SP_DIR%\torch\lib
     rmdir /s /q %SP_DIR%\torch\bin
```

Comment:
Just realized that we need to modify this bit now to actually install the remaining libraries. I need to jump now, will do it in ~2-3h otherwise.